backend/src/test/java/com/rota/database/DbHandlerTests.java

```diff
@@ -0,0 +1,137 @@
+package com.rota.database;
+
+import com.rota.database.orm.engagement.Engagement;
+import com.rota.database.orm.engagement.EngagementRepository;
+import com.rota.database.orm.engagement.EngagementType;
+import com.rota.database.orm.staff.Role;
+import com.rota.database.orm.staff.Staff;
+import com.rota.database.orm.staff.StaffRepository;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.test.annotation.DirtiesContext;
+
+@EnableJpaRepositories
+@DataJpaTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
```

Comment:
If this is how we get around having a pre-populated database before tests, that's okay for now.

In the apps I've built we have 2 different directories which create the sql tables. 1 used for prod/tests, and 1 used for dev. Only the Dev one contains seed data, which keeps it away from the tests. The downside is that if we make changes to 1 script we have to change it in 2 places, so it's not a perfect solution.